src/com/android/settings/applications/managedomainurls/InstantAppWebActionPreferenceController.java

```diff
@@ -37,7 +37,7 @@ public int getAvailabilityStatus() {
 
     public boolean isChecked() {
         return Settings.Secure.getInt(mContext.getContentResolver(),
-                Settings.Secure.INSTANT_APPS_ENABLED, 1) == 1;
+                Settings.Secure.INSTANT_APPS_ENABLED, 0) == 1;
```

Comment:
Seems to be redundant, since `getAvailabilityStatus` will always return `UNSUPPORTED_ON_DEVICE`